myaws/acm_ls.go

```diff
@@ -0,0 +1,89 @@
+package myaws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/acm"
+	"github.com/pkg/errors"
+)
+
+// ACMLsOptions customize the behavior of the Ls command.
+type ACMLsOptions struct {
+	Verbose bool
+	Pending bool
+}
+
+// ACMLs describes Amazon Certificate resources.
+func (client *Client) ACMLs(options ACMLsOptions) error {
+	certificates, err := client.listsCertificate()
+	if err != nil {
+		return errors.Wrap(err, "listsCertificate failed:")
+	}
+	for _, cert := range certificates {
+		if options.Pending && !haveAnyPending(cert) {
+			continue
+		}
+
+		if options.Verbose {
+			fmt.Fprintln(client.stdout, cert.GoString())
+		} else {
+			fmt.Fprintln(client.stdout,
+				cert.NotAfter,
```

Comment:
Can we use `client.FormatTime(cert.NotAfter)` ?
This helper function is used other sub commands and changes output when `--humanize` flag is set to `true` or `false`.

https://github.com/minamijoyo/myaws/blob/d74b3a4f2956c1f146f6ece05c913b894602427b/myaws/time.go#L9-L28